server/handlers.go

```diff
@@ -175,6 +175,7 @@ func (s *Server) discoveryHandler() (http.HandlerFunc, error) {
 		Claims: []string{
 			"aud", "email", "email_verified", "exp",
 			"iat", "iss", "locale", "name", "sub",
+			"preferred_username",
```

Comment:
Is it proper to say that it's supported if only a subset does actually provide it...? 🤔

Reply:
TBH, I have implemented it for these 3 providers since I have experience with that only ;) But I'm happy to extend the implementation. Should we remove `preferred_username` from the discovered claims until that?

Reply:
Just a note: I would probably add a column to the connector list table in the readme.

Reply:
Should I just remove `preferred_username` from the list in the meantime, until we get all connectors to implement this @srenatus ?

Reply:
Since `groups` (which also isn't supported by every provider) is also not in this list, I guess it makes more sense to remove it. But adding a column to the list in the README is still a good idea IMO.

Reply:
all right! resolved :)